web/themes/custom/pineapple/components/_comments.twig

```diff
@@ -83,10 +83,10 @@
     indicator here would break the render cache.
   #}
   <mark class="hidden new-label" data-comment-timestamp="{{ new_indicator_timestamp }}"></mark>
-
+  {% set administrator = bamboo_has_role('administrator', author_id) ? 'TRUE' : 'FALSE' %}
```

Comment:
Looks like ```bamboo_has_role('administrator', author_id)``` already returns a bool value by default, so you don't need to set true or false yourself. You should remove that part and then on line 89 remove "== 'TRUE'"

Reply:
Done.